Source/Spark.swift

```diff
@@ -24,31 +24,126 @@ import UIKit
 
 
 /// The main entry point into the SDK. Contains methods for initializing the SDK and verifying developer and end user credentials.
-open class Spark {
+public class Spark {
```

Comment:
That should be doable! I can certainly see a more complex application using multiple Spark instances for on different accounts to do different things, perhaps monitoring multiple chat rooms as different logical users. I think there might be some problems with multiple media sessions, as there are some singleton aspects of that in the binary libraries, but this is at least a few big steps in that direction. There are a few things that the user would need to do, including using a non-default storage mechanism to differentiate between accounts, but that's part of the reason we made those injectable. 

Apart from that though, it also allows much easier management of the lifecycle. I could drop the entire Spark instance and create a new one without worrying (quite justifiably!) if the old one was going to interfere.